src/eth/command/network/add.ts

```diff
@@ -115,9 +125,9 @@ export const handler = async (argv: Arguments) => {
           ipAddress: ipAddress,
         }
 
-        const spinner = ora('Quorum Network Add ...').start()
+        const spinner = ora(`${networkTypeWithBigFirstLetter} Network Add ...`).start()
         await network.addMemberRemote(addMemberRemoteConfig)
-        spinner.succeed(`Quorum Network Add Member ${memberAddress} Successfully!`)
+        spinner.succeed(`${networkTypeWithBigFirstLetter} Network Add Member ${memberAddress} Successfully!`)
       }
       // TODO: addMemberRemote
```

Comment:
Can delete this TODO (if remote mode use kubernetes)

Reply:
removed